### PR TITLE
Add validator security settings

### PR DESCRIPTION
**Pull request by yorhodes**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -125,6 +125,7 @@
 * [Glossary](resources/glossary.md)
 * [Contract addresses](resources/addresses.md)
 * [Domain identifiers](resources/domains.md)
+* [Validators](resources/validators.md)
```
Comment (nambrot) [inline SUMMARY.md]: Optional: I'd maybe rename this to Default ISM configuration

File: resources/validators.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+description: Mailbox default security settings
+---
+
+# Mailbox default security settings
+
+## Celo
+
+### Validators
+
+| Validator                                | Address                                      | Storage Location                                                                             |
```
Comment (nambrot) [inline resources/validators.md]: Optional: I think the storage locations are probably unnecessary

**Review by nambrot — APPROVED** [review]
🙏

File: resources/validators.md (hunk 2 above)
Comment (yorhodes) [reply resources/validators.md]: why not include?

Comment (yorhodes) [reply resources/validators.md]: agree "unnecessary" but maybe useful?

**Pull request merged**